client_test.go

```diff
@@ -180,37 +180,107 @@ func TestExtensibleTemplates(t *testing.T) {
 
 	// Create a new client with a path to the extensible templates
 	client := bosonFunc.New(
-		bosonFunc.WithTemplates("testdata/templates"),
+		bosonFunc.WithTemplates("testdata/repositories"),
```

Comment:
This will be renamed to "packages" now that we have decided on a rename of --tempates per #381

Reply:
Opeining this as a separate issue, as it has a fair bit of discussion around it.